src/applications/vaos/tests/covid-19-vaccine/components/SelectDate1Page.unit.spec.js

```diff
@@ -323,16 +323,16 @@ describe('VAOS vaccine flow: SelectDate1Page', () => {
     expect(screen.history.push.called).not.to.be.true;
   });
 
-  // Test failure: https://github.com/department-of-veterans-affairs/va.gov-team/issues/84370
-  it.skip('should fetch slots when moving between months', async () => {
+  it('should fetch slots when moving between months', async () => {
     mockEligibilityFetches({
       facilityId: '983',
       typeOfCareId: TYPE_OF_CARE_ID,
       clinics,
     });
 
-    const preferredDate = moment();
-    const slot308Date = moment()
```

Comment:
FYI the root cause of the problem is the `slot308Date` is computed as the `preferredDate` + 1 day. If `preferredDate` (i.e. current day) is the last day of the month, `slot308Date` is already a different month and `secondSlotDate`, which is computed as an offset from `slot308Date`, is three months after than `preferredDate`. This breaks the expectation on lines L397-L405. 

Instead of adding more logic to handle the special case of "last day of the month", I think it's easier to just fix the preferred date as the 15th and avoid all this trouble since it doesn't really reduce our functional coverage.

Reply:
I like that solution!